### DISCUSSION TITLE
How to disable select Template from Elementor Templates "Library" Panel when create new Template?

### DISCUSSION THREAD

**Discussion by datgausaigon** [elementor/elementor#19555] [Developers Community]
When we create new Template, Elementor will let's we choice Template in Library.
How to disable it? Only focus go to design.

Please view Image below:

[https://ibb.co/rdGSJCZ](https://ibb.co/rdGSJCZ)

Thanks

**Comment by NFT-Generator-iMintify** [comment]
Already found a solution for this?
